label_studio/data_import/api.py

```diff
@@ -523,3 +523,40 @@ def get(self, request, *args, **kwargs):
         response['X-Accel-Redirect'] = redirect
         response['Content-Disposition'] = 'attachment; filename="{}"'.format(filepath)
         return response
+
+
+class PresignStorageData(DownloadStorageData):
+    def get(self, request, *args, **kwargs):
+        """ Get export files list
+        """
+        request = self.request
+        task_id = kwargs.get("task_id")
+        fileuri = request.GET.get('uri')
```

Comment:
Changed to `fileuri` to be consistent with the file-proxy and download storage endpoints. Since all 3 of these endpoints are not part of the api, I am using the same approach the previous 2 views were taking.